src/scss/_placeholders.scss

```diff
@@ -90,13 +90,12 @@
 		}
 
 		p {
-			@include oEditorialTypographyBody();
```

Comment:
I think this means we can remove o-editorial-typography from bower.json now

Reply:
let's keep it. Introducing dependencies is a breaking change and I anticipate we may need editorial again in the figure